src/Game/UI/Gumps/OptionsGump.cs

```diff
@@ -3311,6 +3311,23 @@ private void BuildContainers()
             button.MouseUp += (sender, e) => { ContainerManager.BuildContainerFile(true); };
             rightArea.Add(button);
 
+            var openUiManagerGump = new NiceButton
+            (
+                startX + button.Width + 20, startY, 130, 30,
+                ButtonAction.Activate, ResGumps.OpenUIManager
+            )
+            {
+                ButtonParameter = -1,
+                IsSelectable = true,
+                IsSelected = true
+            };
+
+            openUiManagerGump.MouseUp += (sender, e) =>
+            {
+                UIManager.Add(new UiManagerGump());
```

Comment:
I think this could have its own options page/category instead of spawning a new gump? It doesn't make much sense to be in the containers group anyway.

Reply:
Yeah - on Outlands I renamed "containers" to "interface" because we have more options in there. It doesn't make sense for vanilla ClassicUO currently.